tests/prompt_strategies/messages/test_chat.py

```
@@ -1,16 +1,23 @@
-"""
+"""Module for testing chat message internals."""
 
-# pylint: disable=duplicate-code
-import logging
+import os
 import unittest
 
+from transformers import AutoTokenizer
+
+from axolotl.core.chat.messages import (
+    ChatFormattedChats,
```

Ruff (0.11.9)

`axolotl.core.chat.messages.ChatFormattedChats` imported but unused

Remove unused import

(F401)